cipd_packages/codesign/lib/src/file_codesign_visitor.dart

```diff
@@ -34,6 +36,10 @@ class FileCodesignVisitor {
     required this.codesignAppstoreIDFilePath,
     required this.codesignTeamIDFilePath,
     this.dryrun = true,
+    @visibleForTesting this.retryOptions = const RetryOptions(
+      maxAttempts: 5,
```

Comment:
nit: umm do we need 5 attempts, maybe 3 is enough but its really nit

Reply:
3 was the previous number, so increasing to 5 with exponential back off should prevent another breakage.

Reply:
umm I am not sure if retry is the culprit but I am good with adding more retries.

test below might need to be updated from 3 to 5 too. otherwise LGTM